lib/langchain/active_record/hooks.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Langchain
+  module ActiveRecord
+    #
+    # This module adds the following functionality to your ActiveRecord models:
+    # * `vectorsearch` class method to set the provider
+    # * `similarity_search` class method to search for similar texts
+    # * `index_to_vectorsearch` instance method to index the text to the provider
+    #
+    # Usage:
+    #     class Product < ActiveRecord::Base
+    #       vectorsearch provider: Langchain::Vectorsearch::Weaviate.new(
+    #                    api_key: ENV["WEAVIATE_API_KEY"],
+    #                    url: ENV["WEAVIATE_URL"],
+    #                    index_name: "Recipes",
+    #                    llm: Langchain::LLM::OpenAI.new(api_key: ENV["OPENAI_API_KEY"])
+    #                 )
+    #
+    #       after_create :index_to_vectorsearch
+    #       after_update :index_to_vectorsearch
```

Comment:
`after_save` covers both of these, and is one less line.

```suggestion
    #       after_save :index_to_vectorsearch
```